libraries/AP_Airspeed/AP_Airspeed_MS4525.cpp

```diff
@@ -156,16 +156,19 @@ void AP_Airspeed_MS4525::_collect()
       are generated when the bottom port is used as the static
       port on the pitot and top port is used as the dynamic port
      */
-    float diff_press_PSI = -((dp_raw - 0.1f*16383) * (P_max-P_min)/(0.8f*16383) + P_min);
+    float diff_press_PSI  = -((dp_raw - 0.1f*16383) * (P_max-P_min)/(0.8f*16383) + P_min);
```

Comment:
@tridge san. Sorry for the incomprehensible sentences.
@magicrub san, thank you for the comment. @magicrub comment and agreement.
It is an image like the following.

#define DIFF_PRESS(dprow) (-(((dprow) - 0.1f*16383) * (P_max-P_min)/(0.8f*16383) + P_min) )
:
float diff_press_PSI = DIFF_PRESS(dp_row);
float diff_press_PSI2 = DIFF_PRESS(dp_row2);